OctopusProjectBuilder.TestUtils/Properties/AssemblyInfo.cs

```diff
@@ -30,6 +30,6 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-// [assembly: AssemblyVersion("2.7.0.0")]
-[assembly: AssemblyVersion("2.7.0.0")]
-[assembly: AssemblyFileVersion("2.7.0.0")]
+// [assembly: AssemblyVersion("2.8.0.0")]
+[assembly: AssemblyVersion("2.8.0.0")]
```

Comment:
2.8.0.0 is current version on TeamCity. Should it be changed 2.9.0.0 ( as will be substituted from version.json ) ?